documentation/blog/2025-08-14-how-goose-rebuilt-my-website/index.md

```diff
@@ -0,0 +1,71 @@
+
+> Create a 404 page that tells a creative story about GitHub user @taniashiba using their public GitHub data — commit history, contribution graph, repos, or anything else you can access.
+
+I took this prompt, and modified it a bit. My personal GitHub profile and contributions are great, but I also wanted to make sure goose referenced my LinkedIn, Instagram, and all other social channels to get a good grasp of me.
+
+> You can also reference Tania's Bluesky/Instagram/TikTok/Twitter accounts (username @taniashiba) along with her LinkedIn for additional info on her.
+
+![404 page](404page.png)
+
+Quickly brought to life is a 404 page that is witty and actually pretty reflective of who I am and what I like in its styling. It even inserted a shrimp joke I had made in a past social post. I didn't even tell goose about any color combos I love, and somehow it made something that spoke to me. This lit a bright spark of inspiration. So, I asked goose:
+
+> Can you remember that you made this? I'd love for you to make a website in this exact styling for taniachakraborty.com.
+
+## Implementation
+
+goose was helping me resolve what felt like an age-old problem, and made it so simple to do. I gave goose my website, told goose it was hosted on Neocities, and it went to work. After creating general pages with the styling it made for the 404 page, implementation was easy:
+
+1. **Upload** the files goose made onto Neocities
+2. **Review** the site and ask goose to edit or create any pages I needed
+3. **Write content** to fill the different pages of my website (my favorite part)
+
+Then boom, my website was done. No wrestling ancient memories from my mind to relearn CSS, no debugging issues caused by a responsive hover effect I thought was cool at 2 AM, no troubles at all. Goose handled everything. It started with a simple structure, used the styling it came up with from the 404 day challenge, and made changes as I asked for them in conversation. My website went from being embarrasingly empty to being wonderfully designed and easy to edit in *minutes*.
+
+## Always Use Git
+
+It honestly felt like playing a video game, because I could see changes happen live in my local preview, and with git I could save progress as we went. Goose even suggested I add my blog posts from my dev.to page, and created a simple template for me. And if anything wasn't showing up as intended? We just troubleshooted together by me sending screenshots of what I was seeing, and goose simply fixing it.
+
+## A HUGE Timesaver, Seriously
+
+Something that would've taken me anywhere from a week to a month to build, was done instantly. Instead of worrying about learning or relearning a skill in order to build something you need right now, you can learn as you go along with goose. This entire experience reminded me why I fell in love with styling websites for others in the first place, you really create something out of nothing!
```

Comment:
```suggestion
Something that would've taken me anywhere from a week to a month to build, was done instantly. Instead of worrying about learning or relearning a skill in order to build something you need right now, you can learn as you go along with goose. This entire experience reminded me why I fell in love with styling websites for others in the first place. You really create something out of nothing!
```